### PR TITLE
[Video] Lexicon implementation

### PR DESCRIPTION
**Pull request by mozzius**
Stacked on #4732 

Adapts the video player to use the `AppBskyEmbedVideo` lexicon, including aspect ratios and alt text

### CODE REVIEW

File: src/view/com/util/post-embeds/VideoEmbed.tsx [hunk 1]
```diff
@@ -1,50 +1,79 @@
...
+            size="large">
+            <PlayIcon width={48} fill={t.palette.white} />
+          </Button>
+        </>
       )}
     </View>
   )
 }
+
+function clamp(value: number, min: number, max: number) {
```
Comment (haileyok) [inline src/view/com/util/post-embeds/VideoEmbed.tsx:77]: there's a `clamp` helper in `'lib/numbers' btw

File: src/view/com/util/post-embeds/VideoEmbedInner.tsx [hunk 2]
```diff
@@ -67,16 +73,30 @@ export function VideoEmbedInner({}: {
...
       collapsable={false}>
       <VideoView
         ref={ref}
         player={player}
         style={a.flex_1}
+        contentFit="contain"
         nativeControls={true}
+        accessibilityIgnoresInvertColors
+        accessibilityLabel={
+          embed.alt ? _(msg`Video: ${embed.alt}`) : _(msg`Video`)
```
Comment (haileyok) [inline src/view/com/util/post-embeds/VideoEmbedInner.tsx:97]: hmm, yea i guess we need to stick `video:` in front of this huh... might be weird to hear "Video: Video of a cat eating a tuna" but might just be the way it is. We can tweak later if we want after testing VO/TB with the final thing.

**Review by haileyok — APPROVED** [review]
lgtm

File: src/view/com/util/post-embeds/VideoEmbedInner.tsx (hunk 2 above)
Comment (mozzius) [reply src/view/com/util/post-embeds/VideoEmbedInner.tsx:97]: Yeah really just guessing here, might be good to ask for feedback after release

**Pull request merged**